### PR TITLE
Update some training configs to match Data V2

### PR DESCRIPTION
**Pull request by ZhaofengWu**

### CODE REVIEW

File: training_config/coref_spanbert_large.jsonnet [hunk 1]
```diff
@@ -85,10 +85,12 @@ local span_pair_embedding_dim = 3 * span_embedding_dim + feature_size;
...
     "inference_order": 2
   },
-  "iterator": {
-    "type": "bucket",
-    "sorting_keys": [["text", "tokens___token_ids"]],
-    "batch_size": 1
+  "data_loader": {
+    "batch_sampler": {
+      "type": "bucket",
+      "sorting_keys": [["text", "tokens___token_ids"]],
```
Comment (matt-gardner) [inline training_config/coref_spanbert_large.jsonnet]: Remove this line.

Comment (ZhaofengWu) [reply training_config/coref_spanbert_large.jsonnet]: https://github.com/allenai/allennlp/pull/3856#issuecomment-592147896

Comment (matt-gardner) [reply training_config/coref_spanbert_large.jsonnet]: Oh, the span field might actually be longer than the text field here, so yeah, ok, we should keep this.

Comment (matt-gardner) [reply training_config/coref_spanbert_large.jsonnet]: Putting in a comment about that would be good.

**Pull request merged**